src/mars.d

```diff
@@ -1810,105 +1810,105 @@ private void setDefaultLibrary()
  */
 private void addDefaultVersionIdentifiers()
 {
-    VersionCondition.addPredefinedGlobalIdent("DigitalMars");
+    VersionCondition.addAnyGlobal("DigitalMars");
```

Comment:
`addPredefinedGlobalIdent` is for identifiers that are defined before it starts compiling each module. It seems apropos to me. I'd like to avoid more grand renamings. A large part of this PR is that rename, and it obscures what the PR is actually about.